packages.json

```diff
@@ -23001,5 +23001,19 @@
     "description": "NWSync Repository Management utilities",
     "license": "MIT",
     "web": "https://github.com/Beamdog/nwsync"
+  },
+  {
+    "name": "nim-ws",
```

Comment:
This is incorrect, it should be `ws` but `nim-ws` already exists: https://github.com/treeform/ws.

You'll need to pick a different name, I'll revert this.

Reply:
Why this forbidden prefix `nim-` isn't checked by our CIs?

I always forget to check this manually.

Reply:
probably because there are some old packages that have hyphens in the name, it's time we just get rid of these and add the check to package_scanner